modules/ossm-rn-deprecated-features.adoc

```diff
@@ -15,6 +15,25 @@ Deprecated functionality is still included in {product-title} and continues to b
 
 Removed functionality no longer exists in the product.
 
+== Deprecated and removed features {SMProductName} 2.4
```

Comment:
```suggestion
== Deprecated and removed features in {SMProductName} 2.4
```
All of the headings in this section should add `in` in my opinion. WDYT?

It's not worth holding up the merge for, but if you agree, maybe you can write up a Jira to fix all the headings.